webots_ros2_driver/webots_ros2_driver/urdf_spawner.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+# Copyright 1996-2022 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This process simply sends urdf information to the Spawner through a service."""
+
+from launch.actions import ExecuteProcess
+
+
+def get_webots_driver_node(event, driver_node):
+    """Return the driver node in case the service response is successful."""
+    if "success=True" in event.text.decode().strip():
+        return driver_node
+    print("WARNING: the Ros2Supervisor was not able to spawn this URDF robot.")
+    return
+
+class URDFSpawner(ExecuteProcess):
+    def __init__(self, output='screen', name=None, urdf_path=None, translation='0 0 0', rotation='0 0 1 0', normal=False, box_collision=False, init_pos=None, **kwargs):
+        command = [
+                'ros2',
+                'service',
+                'call',
+                '/spawn_urdf_robot',
+                'webots_ros2_msgs/srv/SetWbURDFRobot',
```

Comment:
This naming looks odd... Can't we move the Wb part?
```suggestion
                'webots_ros2_msgs/srv/SetWbURDFRobot',
```

Reply:
That is the way to indicate the type of the srv used. [See an example here](https://docs.ros.org/en/foxy/Tutorials/Topics/Understanding-ROS2-Topics.html#ros2-topic-pub)

Reply:
Yes, but I meant the name of this srv is odd to me.

Reply:
Ah ok I understand now, as discussed together I will rename the msgs to remove the `Wb` (the name of the package `webots_ros2_msgs` is sufficient to show the Webots specificity of these messages ).